README.md

```diff
@@ -141,17 +141,19 @@ String (Default: amd64)
 Specify architecture. Architecture other than the default amd64 can be specified with the `TFENV_ARCH` environment variable
 
 ```console
-TFENV_ARCH=arm tfenv install 0.7.9
+$ TFENV_ARCH=arm
+$ tfenv install 0.7.9
 ```
 
 ##### `TFENV_AUTO_INSTALL`
 
-String (Default: true)
+Boolean (Default: true)
```

Comment:
Bash doesn't have booleans.. this is treated as a boolean, but is still a string